.editorconfig

```diff
@@ -18,17 +18,10 @@ max_line_length = 160
 
 insert_final_newline = true
 
-[*.{csproj,vbproj}]
+[*.{csproj,vbproj,props}]
 insert_final_newline = false
-charset = utf-8-bom
-
-[project.json]
-insert_final_newline = false
-charset = utf-8-bom
-
-[*.nuget.targets]
-insert_final_newline = false
-charset = utf-8-bom

```

Comment:
These files no longer exist (well, there are .nuget.g.targets files in the obj directory, but they're not meant to be edited manually anyway)